src/System.Collections.Immutable/src/Configurations.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netstandard;
```

Comment:
Don't we also need a netcoreapp for Immutable given that is part of NETCore.App?